### ISSUE TITLE
Support __weak

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#111]
Example:

```
@implementation MyClass {
   __weak id _delegate;
}

@end
```

**Comment by drkameleon** [comment]
How is that to be handled? Suggestions? Am I to simply ignore it?

**Comment by charlieMonroe** [comment]
```
class MyClass {

    weak var _delegate: AnyObject?

}
```

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.

----

**Example:**
```objective-c
@implementation MyClass {
    id someVar;
   __weak id _delegate;
}

@end
```

**Output:**
```swift
class MyClass {

	var someVar: AnyObject?
	weak var _delegate: AnyObject?
	
}
```
